graphql2/schema.graphql

```diff
@@ -543,6 +543,25 @@ type Alert {
 
   # Escalation Policy State for the alert.
   state: AlertState
+
+  # recent log entries for the alert.	
+  recentEvents(input: AlertRecentEventsOptions): AlertLogEntryConnection!
+}
+
+input AlertRecentEventsOptions {
+  limit: Int = 50
```

Comment:
`DefaultMaxResults` that we're using is 15, we should reflect that here, or perhaps better, omit the "default" since they are all the same. That way the schema doesn't get out of date if we change `DefaultMaxResults`.